tools/run_tests/python_utils/report_utils.py

```diff
@@ -24,9 +24,6 @@
 import string
 import xml.etree.cElementTree as ET
 
-import six
```

Comment:
No need to work on the Python scripts under `tools/run_tests`. This is just infrastructure code and, more problematically, there aren't good unit tests for it.

Reply:
Removed changes to `tools/*`.